helper/benchhelpers/benchhelpers.go

```diff
@@ -0,0 +1,19 @@
+package benchhelpers
+
+import (
+	"testing"
+
+	testing2 "github.com/mitchellh/go-testing-interface"
+)
+
+type tbWrapper struct {
+	testing.TB
+}
+
+func (b tbWrapper) Parallel() {
+	// no-op
+}
+
+func TBtoT(tb testing.TB) testing2.T {
```

Comment:
This felt a bit odd at first, but I see why you're doing it since `go-testing-interface` explicitly only supports satisfying `testing.T` as [mentioned here](https://github.com/mitchellh/go-testing-interface/pull/5#issuecomment-624214115).

Reply:
Yeah. This wasn't my favorite solution, but I don't see a better way. :/